recipes/cppcheck/all/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 2.8.12)
+project(cmake_wrapper)
+
+include("conanbuildinfo.cmake")
+conan_basic_setup(TARGETS)
+
+# Set include paths
+conan_global_flags()
+
+# Setup libraries to link
+set(Z3_LIBRARIES CONAN_PKG::z3)
+set(PCRE_LIBRARY CONAN_PKG::pcre)
```

Comment:
scratch that. this won't work as you pointed out, because find_library won't guess the dependencies. Too bad the upstream project uses cmake variables and find_library, instead of cmake targets :/